OfficeDev/office-ui-fabric-react
# 5.0 selection fix from fabric 6 commit of a2d5591

#### Pull request checklist

- [v] Addresses an existing issue: Fixes manually pick up fabric 6 fix of a2d5591
- [v] Include a change request file using `$ npm run change`

#### Description of changes

Ensure SelectZone set selection before opening the ContextMenu

#### Focus areas to test

Menu test
1) click dotdotdot the current item should be selected and contextMenu popup
2) right click on DisplayColumn item selected then contextMenu popup
3) click on heroCommand on namefield, item should also be selected

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/7389)
